### PR TITLE
Created objects for teams, matchups and teams

### PR DESCRIPTION
**Pull request by jbbuckner**

### CODE REVIEW

**Comment by jbbuckner** [comment]
@stevehansell created the pull request with everything updated so far.

**Comment by stevehansell** [comment]
@jbbuckner Sweet. Update the description at the top of the pull request and describe what you did, how to test your changes, what I should expect when I run the code and anything I should take a closer look at (if any).

File: public/js/matchup.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	CreateMatchups: function(totalRounds){
+		var matchups = [];
+		//create a matchup
+		//put highest seeded team obj in
+		//put lowest seeded team obj in
+		//need round?
+		//columns = ((Math.log(Bracket.numberofTeams)/Math.log(2))+1);
+		for (var i=0; i<totalRounds; i++){
+			var seedsInColumn = Math.pow(2,i);
+			switch (seedsInColumn){
```
Comment (stevehansell) [inline public/js/matchup.js]: @jbbuckner I'm not sure if you need a switch statement here. I think the for-loop in the default case will do exactly what you're doing. Try that out and see if you get the same result.

**Comment by jbbuckner** [comment]
@stevehansell this should now create the correct number of matchups for each round and the final round should have the correct matchups in no order. If you choose 32 teams there should be 32 matchups the last matchup should be between the 2 middle teams 16 and 17 in round 6.

**Comment by stevehansell** [comment]
@jbbuckner Just to clarify, the teams should not be seeded correctly in the match-ups?

File: public/js/matchup.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+var Matchup = {
+	CreateMatchups: function(totalRounds, teams){
+		var matchups = [];
+
+		columns = (Math.log(teams.length)/Math.log(2))+1;
```
Comment (stevehansell) [inline public/js/matchup.js]: Needs to be declared with `var`

File: public/js/constructor-2.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
+
```
Comment (stevehansell) [inline public/js/constructor-2.js]: Anything being used here? If not, lets delete the file.

File: public/js/constructor-beta.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
+var classID="seed-";
```
Comment (stevehansell) [inline public/js/constructor-beta.js]: Same here: lets delete if not being used.

**Comment by jbbuckner** [comment]
@stevehansell the teams are seeded correctly, but the matchups are not sorted.

**Comment by stevehansell** [comment]
@jbbuckner Wouldn't seed 1 be a higher seed than 2?

screen shot 2014-08-13 at 3 17 24 pm

**Comment by stevehansell** [comment]
@jbbuckner Functionally, it looks good. One thing I wonder about is the `round` value. Do you think the  initial round should be the higher number, decreasing until the final round where it equals `1`?

I'm typically a style-nazi when it comes to code, so I'm going to suggest [reading this page on code conventions](http://javascript.crockford.com/code.html). Ultimately, this is your project so you can define how you want to format things, but following a general convention makes it easier for others to review and contribute.

Merging to master. :rocket: :moon:

**Pull request merged**